unattended_installer/install_functions/installCommon.sh

```diff
@@ -84,6 +84,29 @@ function installCommon_addWazuhRepo() {
 
 }
 
+function installCommon_aptInstall() {
+
+        i=0
```

Comment:
```suggestion
    i=0
```
Review the indentation

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1401/commits/fccfc5546477826ac31928bc1caf68c0531d9976